Build fails on Xcode 7.3 beta 2

just a heads up, Whisper fails compilation on Xcode 7.3 beta 2.

```
ExampleApp/Pods/Whisper/Source/WhisperFactory.swift:289:72: error: value of type 'Index' (aka 'Int') has no member 'value'
        performControllerMove(navigationController.viewControllers[Int(index.value) - 1])
```

--- a/Source/WhisperFactory.swift
+++ b/Source/WhisperFactory.swift
@@ -301,7 +301,7 @@ extension WhisperFactory: UINavigationControllerDelegate {
 
       if let index = navigationController.viewControllers.indexOf(viewController) where index > 0 {
         edgeInsetHeight = -WhisperView.Dimensions.height
-        performControllerMove(navigationController.viewControllers[Int(index.value) - 1])
+        performControllerMove(navigationController.viewControllers[Int(index) - 1])
         break
       }
     }

